### ISSUE TITLE
With Keyboard naviagtion, 'Toggle between mobile and non-mobile mode' button and 'code sample' region are not properly visible at 320px width

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#563]
<dl><dt>Description</dt><dd>Content is not properly visible at 320px width

While navigating through keyboard after 'Navitem5' element keyboard focus goes on below controls one after another which is not properly visible at 320px width by following the same navigation steps - 
Elements : 
1. Toggle between Mobile and Non-Mobile mode
2. Code sample region</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>Content should be visible and accessible to the user with keyboard at 320Px width. </dd><dt>Source code</dt><dd><pre><code><button type="button" class="NavBarButton Button--primary">Toggle between mobil and non-mobile mode</button></code></pre></dd><dt>Checkpoint</dt><dd>Reflow (1.4.10.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/NavBar</dd></dl>

**Comment by schne324** [comment]
Duplicate of #567
